### PR TITLE
Update hand-curated list of Cell Classes and Subclasses

### PR DESCRIPTION
**Pull request by pablo-gar**
## Reason for Change

see https://czi-sci.slack.com/archives/C024HCSH9PT/p1697232968315399 and https://czi-sci.slack.com/archives/C024HCSH9PT/p1700261725188729

## Changes

Expands list of Cell Classes and Sublclasses to reduce number of orphan cell types

see  https://docs.google.com/spreadsheets/d/1ntR-yK2i6k1iTpejosy1ymUxwD-BCYvWgQPXG8DgVf0/edit#gid=0

### CODE REVIEW

File: scripts/compute_mappings/compute_tissue_and_cell_type_mappings.py [hunk 1]
```diff
@@ -26,7 +26,7 @@
 # The descendant mappings should be updated when:
 #
 # 1. The ontology version is updated,
-# 2. A new tissue or cell type is added to the production corpus, or,
+# 2. A tissue or cell type is added to the production corpus, or,
```
Comment (pablo-gar) [inline scripts/compute_mappings/compute_tissue_and_cell_type_mappings.py:29]:
```suggestion
# 2. A new tissue or cell type is added to the production corpus, or,
```

File: scripts/compute_mappings/compute_tissue_and_cell_type_mappings.py [hunk 2]
```diff
@@ -219,15 +219,29 @@
...
-    "CL_0002494",
-    "CL_0002320",
-    "CL_0000473",
-    "CL_0000066",
-    "CL_0000988",
-    "CL_0000187",
-    "CL_0002319",
-    "CL_0011115",
-    "CL_0000151",
+   "CL_0002494",  # cardiocyte
```
Comment (pablo-gar) [inline scripts/compute_mappings/compute_tissue_and_cell_type_mappings.py:222]:
```suggestion
    "CL_0002494",  # cardiocyte
```

File: scripts/compute_mappings/compute_tissue_and_cell_type_mappings.py [hunk 3]
```diff
@@ -236,35 +250,80 @@
...
-    "CL_0000625",
-    "CL_0000679",
-    "CL_0000617",
-    "CL_0000499",
-    "CL_0000576",
-    "CL_0000101",
-    "CL_0000669",
-    "CL_0000152",
-    "CL_0000100",
+    "CL_0002494",  # cardiocyte
```
Comment (pablo-gar) [inline scripts/compute_mappings/compute_tissue_and_cell_type_mappings.py:253]:
```suggestion
```

**Review by MillenniumFalconMechanic — APPROVED** [review]
LGTM, thanks Pablo!

**Pull request merged**